pjmedia/src/pjmedia/echo_webrtc.c

```diff
@@ -72,6 +74,7 @@ typedef struct webrtc_ec
     unsigned	tail;
     unsigned    clock_rate;
     unsigned	channel_count;
+    unsigned	num_bands;
     unsigned    subframe_len;
     sample      tmp_buf[BUF_LEN];
     sample      tmp_buf2[BUF_LEN];
```

Comment:
perhaps we can apply some optimization to allocate `tmp_buf & tmp_buf2` dynamically, i.e: allocate `(echo->num_bands*NUM_SAMPLES)`, instead of `NUM_BANDS*NUM_SAMPLES`, as usually `echo->num_bands` should be 1 as default clock rate is 16khz?